openfe/setup/methods/openmm/equil_rbfe_methods.py

```diff
@@ -193,6 +196,8 @@ class SamplerSettings(BaseModel):
     online_analysis_minimum_iterations : float
       Set number of iterations which must pass before online analysis is
       carried out. Default 50.
+    n_repeats : int
```

Comment:
So one of the common use case (and a pretty critical one at that) for repeats is that you don't know how many of them are needed _a priori_, that is to say that you'll often run a single repeat look at the result and then go "ok yeah this isn't a terrible result, do N more to get some good statistics".

It's unclear to me, does this model (and it's intended expansions of it) allow for this?

Reply:
So for this, you'd need to alter the Settings, so you'd technically have a different Protocol object (which makes sense, you're running the method differently).  I think what you're describing is covered by the idea of a Strategy box.